src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/RightsManagement/User.cs

```diff
@@ -135,7 +135,7 @@ public override bool Equals(object obj)
 
             ContentUser userObj = (ContentUser)obj;
 
-            return (String.CompareOrdinal(_name.ToUpperInvariant(), userObj._name.ToUpperInvariant()) == 0)
```

Comment:
@lindexi Did you intentionally left "ToUpperInvariant()" here?
If no, then we can update it.

Reply:
Thank you @rchauhan18 and I will fix it.